setup.cfg

```diff
@@ -0,0 +1,16 @@
+[isort]
+include_trailing_comma = True
+indent = '    '
+line_length = 120
+multi_line_output = 3
+known_third_party = codejail
+
+[wheel]
```

Comment:
This belongs together with the setup.py PR

Reply:
I quit the wheel section in 8c0b8b8ada32e5e7b0e15fc1b05f33deea2c503f.
Next: Make the setup PR